normalize/property/types.py

```diff
@@ -57,15 +59,15 @@ def parse_datetime(not_a_datetime):
     },
 )
 LongProperty = make_property_type(
-    "LongProperty", isa=long, trait_name="long",
+    "LongProperty", isa=six.integer_types[-1], trait_name="long",
```

Comment:
I'd really like at least a comment explaining the [dangerous] thing you're doing here.

Reply:
Oh yeah this garbage. Integer_types is a tuple of `(int, long)` in python2, just `(int)` in python3.
https://github.com/benjaminp/six/blob/aa4e90bcd7b7bc13a71dfaebcb2021f4caaa8432/six.py#L50

So I can always know the last one is either long or int depending on the version. Need to cut this RIGHT out the soonest we don't need py2.